osf/models/user.py

```diff
@@ -644,7 +644,23 @@ def osf_groups(self):
         OSFGroup = apps.get_model('osf.OSFGroup')
         return get_objects_for_user(self, 'member_group', OSFGroup, with_superuser=False)
 
-    def is_institutional_admin(self, institution):
+    def is_institutional_admin_or_curator(self, institution=None, node=None):
```

Comment:
This seems like two separate functions: `is_institutional_admin` and `is_curator`. Should it be?

Reply:
My feeling is the word curator only applies when you actually have a node, so all curator are insti admins which makes lumping these together somewhat justifiable, but your right it's unclear. Let me address the other things and come back to this.

Reply:
I split the method up into three separate ones, that looks much better thank you.